sdk/eventhub/Azure.Messaging.EventHubs/Directory.Build.props

```diff
@@ -29,10 +29,7 @@
     <!-- Do not inherit implicit dependencies from the engineering system during build or packaging -->
     <ImportDefaultReferences>false</ImportDefaultReferences>
 
-     <!-- Disable the custom analyzer; it is currently in development and based against draft standards. -->
-     <EnableClientSdkAnalyzers>false</EnableClientSdkAnalyzers>
-
-     <!-- If there was no override specified, assume a project reference to Azure.Core -->
```

Comment:
Slightly orthogonal.  This came up in another discussion and is believed to be a legacy artifact that is no longer necessary.